packages/api-derive/src/bounties/bounties.ts

```diff
@@ -29,23 +30,31 @@ function parseResult (api: ApiInterfaceRx, [maybeBounties, maybeDescriptions, id
 }
 
 export function bounties (instanceId: string, api: ApiInterfaceRx): () => Observable<DeriveBounties> {
-  const bountyBase = api.query.bounties
-    ? api.query.bounties
-    : api.query.treasury;
+  const bountyBase = api.query.bounties ? api.query.bounties : api.query.treasury;
 
   return memo(instanceId, (): Observable<DeriveBounties> =>
-    bountyBase.bountyCount<BountyIndex>().pipe(
-      switchMap(() => bountyBase.bounties.keys()),
-      switchMap((keys): Observable<Result> => {
+    combineLatest([
+      bountyBase.bountyCount<BountyIndex>(),
+      api.query.council.proposalCount<ProposalIndex>()
+    ]).pipe(
+      switchMap(() =>
+        combineLatest([
+          bountyBase.bounties.keys(),
+          api.derive.council.proposals()
+        ])
+      ),
+      switchMap(([keys, proposals]): Observable<Result> => {
         const ids = keys.map(({ args: [id] }) => id as BountyIndex);
+        const bountyProposalCalls = [api.tx.bounties.approveBounty, api.tx.bounties.closeBounty, api.tx.bounties.proposeCurator, api.tx.bounties.unassignCurator];
```

Comment:
This could be either on treasury or on bounties (current dev is bounties, current deployed is treasury)

Reply:
Yes, thank you for that! I was just examining why is that not working in apps and I think that's the case